wherehows-web/app/services/dataset-meta.ts

```diff
@@ -0,0 +1,20 @@
+import Service from '@ember/service';
+import { IObject } from 'wherehows-web/typings/generic';
+import { get } from '@ember/object';
+
+export default class DatasetMeta extends Service {
+  healthScore: number;
+
+  healthScores: IObject<number> = {};
+
+  setHealthScoreForUrn(urn: string, score: number) {
+    get(this, 'healthScores')[urn] = score;
+  }
+}
+
+// DO NOT DELETE: this is how TypeScript knows how to look up your services.
+declare module '@ember/service' {
```

Comment:
Please move this to the typings for services https://github.com/linkedin/WhereHows/blob/master/wherehows-web/app/typings/app/services.d.ts, also no need for the DND comment

Reply:
This is actually an incomplete leftover idea from a failed concept lol. Will delete